lib/slack/sends.ex

```diff
@@ -18,23 +21,16 @@ defmodule Slack.Sends do
   end
 
   def send_message(text, user_id = "U" <> _user_id, slack) do
-    user_name = Slack.Lookups.lookup_user_name(user_id, slack)
-    send_message(text, user_name, slack)
+    send_message_to_user(text, user_id, slack)
   end
 
-  def send_message(text, user = "@" <> _user_name, slack) do
-    direct_message_id = Lookups.lookup_direct_message_id(user, slack)
+  def send_message(text, user_id = "W" <> _user_id, slack) do
+    send_message_to_user(text, user_id, slack)
+  end
 
-    if direct_message_id do
-      send_message(text, direct_message_id, slack)
-    else
-      open_im_channel(
-        slack.token,
-        Lookups.lookup_user_id(user, slack),
-        fn id -> send_message(text, id, slack) end,
-        fn reason -> reason end
-      )
-    end
+  def send_message(text, user = "@" <> _user_name, slack) do
+    user_id = Lookups.lookup_user_id(user, slack)
```

Comment:
Would printing a deprecation warning make sense here too?

Reply:
I thought about it; by adding the deprecation warnings in `lookups.ex` we basically get them for free at this level.